jdaviz/configs/cubeviz/plugins/cube_listener.py

```diff
@@ -0,0 +1,150 @@
+import numpy as np
+from contextlib import contextmanager
+import sys
+import os
+import time
+
+try:
+    from strauss.sonification import Sonification
+    from strauss.sources import Events
+    from strauss.score import Score
+    from strauss.generator import Spectralizer
+    from tqdm import tqdm
+except ImportError:
+    pass
```

Comment:
is it possible for any of these imports to fail but there still be a call to any of the logic below (either manual API calls or if none of the imports in the plugin itself fail and so `_has_strauss = True` but `tqdm` failed to import, for example)?

Reply:
I wonder if we can either put the import `tqdm` in the `else` or import `_has_strauss` from the plugin.  But yes, probably can be a follow-up.

Reply:
I think if we decide to move most of the `cube_listener.py` code to the sonify plugin then we can include this effort in #3330 .

Reply:
@james-trayford do we really need tqdm ?